plugins/modules/idrac_certificates.py

```diff
@@ -222,19 +255,21 @@
 from ansible_collections.dellemc.openmanage.plugins.module_utils.utils import reset_idrac
 
 NOT_SUPPORTED_ACTION = "Certificate {op} not supported for the specified certificate type {certype}."
-SUCCESS_MSG = "Successfully performed the '{command}' operation."
+SUCCESS_MSG = "Successfully performed the '{command}' operation. "
+SUCCESS_MSG_SSL = "Successfully performed the SSL key upload and '{command}' operation. "
```

Comment:
Additional space at the end of line
```suggestion
SUCCESS_MSG_SSL = "Successfully performed the SSL key upload and '{command}' operation."
```